src/store/filter.reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { createReducer, ActionType } from 'typesafe-actions';
+import * as Actions from './actions/filter.action';
+import { Data } from '../pages/comparison/interfaces/filter.interface';
+
+export type ProfileAction = ActionType<typeof Actions>;
+
+type State = Data | null;
+
+const initialState = {
```

Comment:
The initialState interface is missing. Should be `const initState: Data = {...}`

Reply:
already fixed